Book API Homework/bookapi/book/views.py

```diff
@@ -25,4 +25,30 @@ def book_list(request):
         if serializer.is_valid():
             serializer.save()
             return JsonResponse(serializer.data, status=201)
-        return JsonResponse(serializer.errors, status=400)
\ No newline at end of file
+        return JsonResponse(serializer.errors, status=400)
+
+@csrf_exempt
```

Comment:
Maybe we should add comments above the function to give insight about what it does and how it works.